projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-default-expression.component.html

```diff
@@ -33,7 +33,7 @@
     />
 </igx-input-group>
 
-<igx-input-group #inputGroupValues type="box" [displayDensity]="'compact'" [supressInputAutofocus]="true">
```

Comment:
Changing the hardcoded display density to the one from the grid will make this input to be bigger then before, regardless of grid's display density. The dialog itself will have the same height and when you add new filter condition it won't be fully visible. My suggestion is to scroll to the end so the newly added filtering condition will be fully visible.

Reply:
@bkulov I agree that whenever a new filtering condition is added, we should scroll to the bottom/end so both the new filtering condition and the "Add Filter" button are visible.
However since the newly added filtering condition was not fully visible up until now as well, we discussed that we should probably implement the **scrolling to bottom after adding new filtering condition** functionality in separate a issue (If would like, you can create it by including all the information that you think is necessary).